docs/devices-api/api-reference/get-body-data.mdx

```diff
@@ -52,7 +51,18 @@ const response = await metriportClient.getBodyData(
     "height_cm": 189,
     "lean_mass_kg": 43,
     "muscle_mass_kg": 22,
-    "weight_kg": 78
+    "weight_kg": 78,
+    "weight_samples_kg": [
+      {
+        "time": "2022-12-21T19:42:20.000Z",
+        "value": 78,
+        "data_source": {
+          "id": "1234",
+          "name": "Body Cardio",
+          "type": "Scale"
+        }
+      }
+    ]
```

Comment:
we'll also have to update the model example: https://github.com/metriport/metriport/blob/1fab200fd1a7dfa9e9279bdccdd09a3dcc905f67/docs/devices-api/more-info/models.mdx#L37-L48